src/emulator/util/include/util/BitField.h

```diff
@@ -0,0 +1,257 @@
+#pragma once
+
+#include "util/types.h"
+#include "log.h"
+
+#define strcat_(x, y) x ## y
```

Comment:
These macros and includes look unnecessary.

Reply:
Oh yea I added them when I was debugging `bf_t` `constexpr` stuff and forgot to take them out.